src/components/ContactPreview.tsx

```diff
@@ -12,12 +12,37 @@ type Props = {
   contact: Profile
 }
 
+export function useStatus() {
+  return {
+    isUserOnline: (contact: Profile) =>
+      contact?.timestamp_last_connection
+        ? Date.parse(new Date().toISOString()) -
+            Date.parse(contact.timestamp_last_connection) / 1000 >
+          60
+        : false,
+    UserStatus: ({ isOnline }: { isOnline: boolean }) => (
+      <div
+        className={css({
```

Comment:
suggestion(ui): If you add a border white 2px, it will look better in gradient backgrounds and with any profile pic

Reply:
nice 🤩